akka-remote/src/test/scala/akka/remote/artery/RemoteSendConsistencySpec.scala

```diff
@@ -62,6 +70,13 @@ abstract class AbstractRemoteSendConsistencySpec(config: Config)
   val addressB = address(systemB)
   val rootB = RootActorPath(addressB)
 
+  private def actorRefBySelection(path: ActorPath) = {
+    system.actorSelection(path) ! Identify(None)
```

Comment:
How about using a correlation id (`Identify(id)` passes the id back in `ActorIdentity(id, ref)`) instead and expecting the exact response(s)?

Reply:
Done